app/controllers/searches_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SearchesController < ApplicationController
+  def index
+    @search_results = Item.search(params[:search]) ||
+                      Review.search(params[:search])
```

Comment:
Use 2 (not 18) spaces for indenting an expression in an assignment spanning multiple lines.